run_bot.sh

```
@@ -1,1082 +1,194 @@
 #!/bin/bash
 
 # Meet Teams Bot - Serverless Runner
-# This script provides an easy way to run the bot in serverless mode
-
 set -e
 
-# Colors for output
+# Colors
 RED='\033[0;31m'
 GREEN='\033[0;32m'
 YELLOW='\033[1;33m'
```

Shellcheck (0.11.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)